### PR TITLE
Improvement/launch

### PR DESCRIPTION
**Pull request by snova-francescar**
Improve the financial assistant for the launch.

- Prepopulating fields.
- New SambaNova credentials.
- Several improvements.

### CODE REVIEW

File: financial_assistant/README.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+We recommend that you run the starter kit in a virtual environment or use a container. We also recommend using Python >= 3.10 and < 3.12.
+
+## Option 1: Use a virtual environment
+
+If you want to use virtualenv or conda environment:
+
+1. Install and update pip.
+
+    ```bash
+    cd ai_starter_kit/enterprise_knowledge_retriever
```
Comment (snova-petrojm) [inline financial_assistant/README.md:90]: This is EKR stuffs. Need to be updated with financial stuffs. Same thing for other parts of the README

File: financial_assistant/README.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    python3 -m venv enterprise_knowledge_env
+    source enterprise_knowledge_env/bin/activate
+    pip  install  -r  requirements.txt
+    ```
+
+2. Run the following command:
+
+    > This kit can be deployed using either a simple LCECL implementation or a LangGraph implementation, both of which produce identical results. This demonstrates the flexibility of the kit and showcases how to leverage these two approaches in Rag applications.
```
Comment (snova-petrojm) [inline financial_assistant/README.md:98]: Same comment here

File: financial_assistant/README.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+NOTE: If you are deploying the docker container in Windows be sure to open the docker desktop application. 
+
+To run the starter kit  with docker, run the following command:
+
+    docker-compose up --build
+
+You will be prompted to go to the link (http://localhost:8501/) in your browser where you will be greeted with the streamlit page as above.
+
+Here's a short video demonstrating docker deployment:
```
Comment (snova-petrojm) [inline financial_assistant/README.md:126]: Here, too. This is a video for EKR.

**Comment by snova-petrojm** [comment]
In Main README, we need to update the link in the table for this kit, Thanks
<td width="20%"><a href="financial_assistants/README.md"> Financial Assistant</a></td>

File: financial_assistant/README.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+https://github.com/sambanova/ai-starter-kit/assets/150964187/4f82e4aa-c9a9-45b4-961d-a4b369be5ec4
+
+## Further settings
+
+For the `SEC-EDGAR` functionalities, company name and email are used to form a user-agent of the form:
+USER_AGENT: ```<Company Name> <Email Address>```
+```
+# Your organization
+SEC_API_ORGANIZATION="<your organization>"
```
Comment (snova-kwasia) [inline financial_assistant/README.md:109]: If we are auto populating this, do we still need this in the readme?

**Review by snova-kwasia — APPROVED** [review]
LGTM @snova-francescar I left a comment about the readme but it's non urgent

File: financial_assistant/README.md (hunk 4 above)
Comment (snova-francescar) [reply financial_assistant/README.md:109]: Yes, because we are only populating that for` prod_mode = True`.

**Pull request merged**